kong/plugins/zipkin/opentracing.lua

```diff
@@ -72,20 +72,19 @@ if subsystem == "http" then
     local tracer = self:get_tracer(conf)
     local req = kong.request
     local wire_context = tracer:extract("http_headers", req.get_headers()) -- could be nil
-    local path_with_query = req.get_path_with_query()
     local method = req.get_method()
-    local url = req.get_scheme() .. "://" .. req.get_host() .. ":"
```

Comment:
Note that this is adding more than just the path - namely, the scheme and the host. So the tags will be `http://foo.bar/baz` instead of just `/baz`.

Reply:
Done!